### ISSUE TITLE
 bufio.Scanner: token too long.

### ISSUE THREAD

**Issue by wolftankk** [apache/rocketmq-client-go#135]
version: native branch

UnitMode: PushConsumer

errmsg:  

```bash
INFO[0000] start a nwe Pull Message task [ConsumerGroup: accountDeposit-qa, Topic: pushOrderToFdc, MessageQueue: 1] for [accountDeposit-qa] 
ERRO[0000] net: 10.100.11.100:10911 scanner exit, Err: bufio.Scanner: token too long. 
INFO[0000] receive broker's notification, the consumer group: accountDeposit-qa 
ERRO[0002] lock mq to broker: 10.100.11.100:10911 error request timeout 
ERRO[0003] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
WARN[0003] do balance for Group: accountDeposit-qa, Topic: %RETRY%accountDeposit-qa get consumer id list failed 
ERRO[0003] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
WARN[0003] do balance for Group: accountDeposit-qa, Topic: pushOrderToFdc get consumer id list failed 
ERRO[0006] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
WARN[0006] do balance for Group: accountDeposit-qa, Topic: %RETRY%accountDeposit-qa get consumer id list failed 
WARN[0010] pull message from 10.100.11.100:10911 error: request timeout 
INFO[0010] pull MessageQueue: 0 sleep 3000 ms for mq: MessageQueue [topic=pushOrderToFdc, brokerName=broker-1, queueId=0] 
WARN[0010] pull message from 10.100.11.100:10911 error: request timeout 
INFO[0010] pull MessageQueue: 1 sleep 3000 ms for mq: MessageQueue [topic=pushOrderToFdc, brokerName=broker-1, queueId=1] 
ERRO[0013] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
WARN[0013] do balance for Group: accountDeposit-qa, Topic: pushOrderToFdc get consumer id list failed 
ERRO[0016] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
WARN[0016] do balance for Group: accountDeposit-qa, Topic: %RETRY%accountDeposit-qa get consumer id list failed 
ERRO[0023] lock mq to broker: 10.100.11.100:10911 error request timeout 
WARN[0023] pull message from 10.100.11.100:10911 error: request timeout 
INFO[0023] pull MessageQueue: 0 sleep 3000 ms for mq: MessageQueue [topic=pushOrderToFdc, brokerName=broker-1, queueId=0] 
WARN[0023] pull message from 10.100.11.100:10911 error: request timeout 
INFO[0023] pull MessageQueue: 1 sleep 3000 ms for mq: MessageQueue [topic=pushOrderToFdc, brokerName=broker-1, queueId=1] 
ERRO[0026] net: 10.100.11.101:10911 scanner exit, Err: bufio.Scanner: token too long. 
ERRO[0029] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
WARN[0029] do balance for Group: accountDeposit-qa, Topic: pushOrderToFdc get consumer id list failed 
INFO[0030] {"clientID":"10.100.1.183@DEFAULT","producerDataSet":[],"consumerDataSet":[{"groupName":"accountDeposit-qa","consumeType":"PUSH","messageModel":"CLUSTERING","consumeFromWhere":"CONSUME_FROM_FIRST_OFFSET","subscriptionDataSet":[{"ClassFilterMode":false,"Topic":"pushOrderToFdc","SubString":"*","Tags":null,"Codes":null,"SubVersion":0,"ExpType":"TAG"},{"ClassFilterMode":false,"Topic":"%RETRY%accountDeposit-qa","SubString":"*","Tags":null,"Codes":null,"SubVersion":0,"ExpType":"TAG"}],"unitMode":false}]} 
ERRO[0032] get consumer list of [accountDeposit-qa] from 10.100.11.100:10911 error: request timeout 
```

**Comment by wolftankk** [comment]
rocketmq version less than 4.5.0, it will be got the error.

**Comment by wolftankk** [comment]
in new version, it will also get the error.

I guess the cause what the msg's content is to large.

**Comment by wenfengwang** [comment]
fixed at: https://github.com/apache/rocketmq-client-go/pull/169

**Comment by ShannonDing** [comment]
closed due to #169.
